### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by rastaman111** [zhiyongzou/iRead#6] [General]
hello, I have been watching your project for a long time and admire it, but I noticed that you are stuck and have not made comments for a long time

**Comment by zhiyongzou** [comment]
> hello, I have been watching your project for a long time and admire it, but I noticed that you are stuck and have not made comments for a long time

Busy with work recently😭

**Reply by vishalpatel-simformsolutions** [reply]
Hey . I have added your library in my project. 
And its working really perfect.
but Now I have to give note down feature.
So Is this ready or not ?
If you have any idea/suggestion about it then please share with me.

Thank you

**Reply by vishalpatel-simformsolutions** [reply]
Also, I have added a note-down mechanism, but it only works on the same device resolutions.
If device resolutions change it change the page number and the whole mechanism is messed up.
Is there any suggestion/idea here ?? @zhiyongzou

**Comment by giovincentromero** [comment]
Hi Sir @zhiyongzou. Thank you for creating iRead. Looking forward to the notes feature. Thanks.
